aws/cerberus/cerberus.tf

```diff
@@ -108,6 +113,7 @@ locals {
 
   remote_state_acct_key = "${coalesce(var.terraform_remote_state_acct_key, "master/${var.stage}/acct")}"
   remote_state_vpc_key  = "${coalesce(var.terraform_remote_state_vpc_key, "master/${var.stage}/shared-vpc")}"
+  alb_protocol          = "${var.alb_target_group_port == 80 ? "HTTP" : "HTTPS" }"
```

Comment:
The protocol should be mapped to a variable, but default to `HTTP` (e.g. `alb_target_group_protocol`).